app/com/arpnetworking/rollups/MetricsDiscovery.java

```diff
@@ -113,13 +118,24 @@ private void updateMetricsSet(final KairosMetricNamesQueryResponse response) {
         return Optional.ofNullable(next);
     }
 
+    private Predicate<String> toPredicate(final List<String> regexList, final boolean defaultResult) {
+        return regexList
+                .stream()
+                .map(r -> "^" + r + "$")
```

Comment:
Ville brought up a good point here.  Lets not mess with the regex in a weird way.  Simplicity is probably better here.  Lets just pass the string through unchanged and not create a weird regex dialect by mucking with it.  If the user wants to anchor, lets let them anchor.